src/core/thread/mlr_manager.cpp

```diff
@@ -67,10 +68,36 @@ MlrManager::MlrManager(Instance &aInstance)
 void MlrManager::HandleNotifierEvents(Events aEvents)
 {
 #if OPENTHREAD_CONFIG_MLR_ENABLE
+    uint8_t                aSequenceNumber = 0;
+    otBackboneRouterConfig config;
+    if (Get<BackboneRouter::Leader>().GetConfig(config))
+    {
+        aSequenceNumber = config.mSequenceNumber;
```

Comment:
The sequence number recording should be also conducted on routers acting as multicast proxies